recipes/openscenegraph/all/conanfile.py

```diff
@@ -130,29 +130,33 @@ def requirements(self):
         if self.options.get_safe("with_asio", False):
             # Should these be private requires?
             self.requires("asio/1.22.1")
-            self.requires("boost/1.81.0")
+            self.requires("boost/1.82.0")
         if self.options.with_curl:
-            self.requires("libcurl/8.0.1")
+            self.requires("libcurl/8.1.2")
         if self.options.get_safe("with_dcmtk"):
             self.requires("dcmtk/3.6.6")
         if self.options.with_freetype:
             self.requires("freetype/2.13.0")
         if self.options.with_gdal:
-            self.requires("gdal/3.4.3")
+            self.requires("gdal/3.5.2")
         if self.options.get_safe("with_gif"):
             self.requires("giflib/5.2.1")
         if self.options.with_gta:
             self.requires("libgta/1.2.1")
         if self.options.with_jasper:
-            self.requires("jasper/2.0.33")
-        if self.options.get_safe("with_jpeg"):
+            self.requires("jasper/4.0.0")
+        if self.options.get_safe("with_jpeg") == "libjpeg":
             self.requires("libjpeg/9e")
+        elif self.options.get_safe("with_jpeg") == "libjpeg-turbo":
+            self.requires("libjpeg-turbo/2.1.5")
+        elif self.options.get_safe("with_jpeg") == "mozjpeg":
+            self.requires("mozjpeg/4.1.1")
         if self.options.get_safe("with_openexr"):
-            self.requires("openexr/3.1.7")
+            self.requires("openexr/3.1.5")
         if self.options.get_safe("with_png"):
-            self.requires("libpng/1.6.40")
+            self.requires("libpng/1.6.39")
         if self.options.with_tiff:
-            self.requires("libtiff/4.5.1")
+            self.requires("libtiff/4.5.0")
         if self.options.with_zlib:
             self.requires("zlib/1.2.13")
```

Comment:
Could you set this to `[>=1.2.11 <2]` range if you modify the recipe again please? Thanks